salt/metalk8s/kubernetes/ca/kubernetes/init.sls

```diff
@@ -5,7 +5,8 @@
 # ================
 #
 # * installed   -> install and advertise as CA server
```

Comment:
```suggestion
# * installed   -> install as CA server
```
This `installed` state is not advertising anymore.

Reply:
Done